llvm/lib/CAS/ObjectStore.cpp

```diff
@@ -98,6 +99,48 @@ Expected<ObjectProxy> ObjectStore::createProxy(ArrayRef<ObjectRef> Refs,
   return getProxy(*Ref);
 }
 
+Expected<ObjectRef>
+ObjectStore::storeFromOpenFileImpl(sys::fs::file_t FD,
+                                   Optional<sys::fs::file_status> Status) {
+  // \p Copy the file into an immutable memory buffer and call \p store on that.
```

Comment:
What are the `\p` in this comment? Normally `\p` means "parameter" in doxygen. Maybe it should be `\c`?  Also, why would "Copy" be marked up here? I don't think there is a Copy method being used.

Reply:
This was leftover that I missed, I've removed it.

Reply:
Oh ok, I misunderstood use of `\p`, I've changed it to `\c`.